be/src/vec/aggregate_functions/aggregate_function_collect.cpp

```diff
@@ -49,6 +49,10 @@ AggregateFunctionPtr do_create_agg_function_collect(bool distinct, const DataTyp
                     AggregateFunctionCollectListData<T, HasLimit>, HasLimit, std::false_type>>(
                     argument_types, result_is_nullable);
         }
+    } else if (!distinct) {
+        return creator_without_type::create<AggregateFunctionCollect<
```

Comment:
Add comment here void -> use in array/map some complex type

Reply:
done!